docsrc/interactivity.nim

```diff
@@ -2,57 +2,125 @@ import nimib
 
 nbInit
 
+template nimibCode*(body: untyped) =
```

Comment:
indeed nimibCode is useful. should we add it do nimib? I do not mind the off-convention naming (nimibCode instead of nbCode), since it is about nimib itself. It could actually start a new convection that this kind of blocks (honestly not sure if there will be others), could be all start with nimib.

Reply:
I don't think it matters that much that it doesn't start with `nb` as it is only useful when showing code blocks with use nimib templates. And it will most likely be you and me who are using it.  So I'm for adding it :+1:  should I add it in this PR or do you want it in a separate one?

Reply:
This one is fine I guess